features/scan/components/PaymentScreen.tsx

```
@@ -54,6 +55,15 @@ export const PaymentScreen = ({ recipient = "ERZASDER" }: PaymentScreenProps) =>
         }
     };
 
+    if (!data) {
+        return <LoadingScreen />
+    }
+    
+    // Use parsed data recipient if available, otherwise fall back to prop
+    const actualRecipient = recipient || data?.recipient;
+
+    const [amount, setAmount] = useState(data?.amount || "0");
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)